flake8_django/checkers/model_fields.py

```diff
@@ -1,13 +1,18 @@
+import django
 from .checker import Checker
 from .issue import Issue
 
+ALLOW_NULL_BOOLEAN = django.VERSION >= (2, 1)
 
 NOT_NULL_TRUE_FIELDS = [
-    'CharField', 'TextField', 'SlugField',
-    'EmailField', 'UUIDField', 'ImageField',
-    'FileField', 'FilePathField', 'URLField'
+    'CharField', 'TextField', 'SlugField', 'EmailField', 'Field',
+    'UUIDField', 'ImageField', 'FileField'
 ]
-NOT_BLANK_TRUE_FIELDS = ['BooleanField']
+NOT_BLANK_TRUE_FIELDS = []
+
+if not ALLOW_NULL_BOOLEAN:
```

Comment:
Could we do the check for django.VERSION here? Just so that we make it more explicit?